src/views/RecIM/views/Admin/index.jsx

```diff
@@ -6,7 +6,7 @@ import GordonLoader from 'components/Loader';
 import GordonSnackbar from 'components/Snackbar';
 import Header from '../../components/Header';
 import styles from './Admin.module.css';
-import { getParticipantByUsername } from 'services/recim/participant';
+import { editParticipantStatus, getParticipantByUsername } from 'services/recim/participant';
```

Comment:
```suggestion
import { getParticipantByUsername } from 'services/recim/participant';
```
I don't think you use `editParticipantStatus` here anymore